<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>valgrind::GenericConfig</class>
 <widget class="QWidget" name="valgrind::GenericConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>475</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Valgrind</string>
     </property>
     <layout class="QGridLayout">
      <item row="7" column="0">
       <widget class="QCheckBox" name="limitErrors">
        <property name="text">
         <string>Limit errors if too many</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="currentTool">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="maxVisibleItems">
         <number>6</number>
        </property>
        <property name="minimumContentsLength">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Display stac&amp;k to depth:</string>
        </property>
        <property name="buddy">
         <cstring>stackDepth</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Valgrind tool:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>E&amp;xtra Parameters:</string>
        </property>
        <property name="buddy">
         <cstring>valgrindParameters</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="valgrindParameters"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>&amp;Valgrind Executable:</string>
        </property>
        <property name="buddy">
         <cstring>valgrindExecutable</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;Maximum frame stack size (bytes):</string>
        </property>
        <property name="buddy">
         <cstring>maxStackSize</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="valgrindExecutable"/>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="stackDepth">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="maxStackSize">
        <property name="maximum">
         <number>224506763</number>
        </property>
        <property name="value">
         <number>2000000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>71</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
